source/gui/__init__.py

```diff
@@ -474,11 +488,17 @@ def __init__(self, frame):
 			# Translators: The label for the menu item to save current settings.
 			item = self.menu.Append(wx.ID_SAVE, _("&Save configuration"), _("Write the current configuration to nvda.ini"))
 			self.Bind(wx.EVT_MENU, frame.onSaveConfigurationCommand, item)
-		if not globalVars.appArgs.secure:

```

Comment:
Removing this line seems like a regression to me?

Reply:
I should have pointed this out beforehand. What's not clear from the diff is that the code part below the removed line is also part of an indented block for exactly the same if check. So this particular if check can be safely removed to make this code part of the previous if block, which is equivalent. It is quite unrelated to this pr though.

Reply:
Oh yep